packages/validators-util/src/isValidName/isValidName.ts

```diff
@@ -0,0 +1,3 @@
+export const VALID_NAME_REGEX = /^([a-zæøåA-ZÆØÅ]+\s)*[a-zæøåA-ZÆØÅ]+$/;
```

Comment:
Husker jeg hadde en liknende regex tidligere for navn og adresser, men den skapte noen problemer. For eksempel samiske navn (både stedsvnavn og egennavn) bruker tegn utenfor det norske alfabetet. Det er sikkert noen svensker bosatt i Norge som også vil bli skuffet over at navnet deres er ugyldig 😄 Irske og skotske familienavn har noen ganger `'` i seg, og skal ikke se bort fra at personer med slektskap derfra er bosatt i Norge og ønsker seg forsikring. Og sikkert en drøss andre navnetradisjoner jeg glemmer.

Jeg husker vi prøvde å lage en regexp som inkluderte tegn fra flere samiske skriftspråk, svensk, og noen spesialtegn. Mulig enden på visa der ble en denylist med "åpenbart ikke brukt i navn", heller enn en allowlist, rett og slett fordi et navn kan være så mangt.

Reply:
Hmm ja, tenkte tanken. Vet ikke hva nice støtter, men det gir jo mening at det er lov siden vi godtar det i dagens løsninger. Har du eksempel på hva en blocklist?